queries/kcl/highlights.scm

```diff
@@ -0,0 +1,166 @@
+(identifier) @variable
+
+; Reset highlighting in string interpolations
+(interpolation) @none
```

Comment:
This does not work in Neovim. (You cannot use upstream queries unchanged; please see CONTRIBUTING.MD.)

Reply:
Thanks for the quick response. I actually wrote the upstream query file myself and tested with Neovim and it seem to work quite well, but I changed it to only use the documented tags.

Reply:
Ah, ok, I was this in the python highlighter and assumed it would have affect maybe for 'unhighlighted' parts of the interpolated expression, but didn't test. Removed the line.

Reply:
Yes, different highlighters work differently, that was my point about queries being editor specific. 

I should probably tighten up CI to no longer accept `@none` captures.